deeplinkdispatch-base/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkUri.java

```diff
@@ -1215,7 +1217,22 @@ private static boolean decodeIpv4Suffix(
      */
     private static String domainToAscii(String input) {
       try {
-        String result = IDN.toASCII(input).toLowerCase(Locale.US);
+        // When this is a template URI it might get to long with all the placeholder definitions
```

Comment:
do you have a number for what the length limit is?

Reply:
256. I'll add it to the comment.